test/test_generation.py

```diff
@@ -134,6 +134,37 @@ def test_llm_generation_asym_qbits(self):
         print(res)
         assert ("!!!" not in res)
 
+    def test_force_to_autoround_format(self):
+        bits = 4
+        group_size = 32
+        autoround = AutoRound(
+            self.model,
+            self.tokenizer,
+            bits=bits,
+            group_size=group_size,
+            sym=True,
+            iters=1,
+            seqlen=2,
+            dataset=self.llm_dataloader,
+        )
+        autoround.quantize()
+        quantized_model_path = "./saved"
+        autoround.save_quantized(output_dir=quantized_model_path, format="auto_awq", inplace=False)
+        device = "auto"  ##cpu, hpu, cuda
+        from auto_round import AutoRoundConfig
+        quantization_config = AutoRoundConfig(
+            # backend="auto_round:awq",
+            use_auto_round_format=True
```

Comment:
could we resue the quantizaiton_config, if it's a AutoRoundConifg, then focce it to autoround format?